src/pages/404.js

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+
+class NotFoundPage extends React.Component {
```

Comment:
I like that you added this, but again, this feels like it should be a separate PR.